test/ci/buildkite/ngtf-cpu_ubuntu-tf2.yaml

```diff
@@ -27,9 +27,9 @@
 
   - command: |
       source /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv/bin/activate 
-      python3 build_ngtf.py --use_tensorflow_2 \
```

Comment:
This yaml file/pipeline was added to test TF2.0 while keeping TF1.15 default. It can be removed now with TF2.2 as default.